src/main/resources/templates/Home.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link th:href="@{/static/css/style.css}" rel="stylesheet"  />
+    <link th:href="@{/styles/cssandjs/main.css}" rel="stylesheet" />
+
+</head>
+<body>
+<div class="topnav">
+    <a class="active" href="#home">Home</a>
+    <a href="#news">News</a>
+    <a href="#contact">Contact</a>
+    <a href="#about">About</a>
+</div>
+
+<div>
+    <button>Explore</button>
```

Comment:
As I mentioned, don't worry about the style right now, just get the data moving next. It looks just fine